sdk/monitor/monitor-query-logs/test.env

```diff
@@ -0,0 +1,20 @@
+# Retrieve these values from an instance in the Azure Portal.
+MONITOR_WORKSPACE_ID=<The "Workspace ID", as given in the Azure portal for your Log Analytics resource>
+
+# Used to authenticate using Azure AD as a service principal for role-based authentication.
+#
+# See the documentation for `EnvironmentCredential` at the following link:
+# https://learn.microsoft.com/javascript/api/@azure/identity/environmentcredential
+AZURE_TENANT_ID=<AD tenant id or name>
+AZURE_CLIENT_ID=<ID of the user/service principal to authenticate as>
+AZURE_CLIENT_SECRET=<client secret used to authenticate to Azure AD>
```

Comment:
should we delete these secrets-based auth vars?

Reply:
done